liferay-echo/liferay-portal
# LPS-195640 Load Miller Columns actions lazily

Motivation
=======
Load Miller Columns actions when the user clicks on the three dots icon
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-195640)

Steps to Verify:
----------------
1. Go to Site Builder -> Pages
1. Click on the three dot icon of one page
1. Check that all the actions continue working
